File: 프로그래머스/1/12977. 소수 만들기/소수 만들기.js
function solution(nums) {
    let result = 0;
    let isPrime = true;
    
    for (let i = 0; i < nums.length - 2; i++) {
        for (let j = i + 1; j < nums.length - 1; j++) {
            for (let k = j + 1; k < nums.length; k++) {
                isPrime = true;
                for (let l = 2; l < nums[i] + nums[j] + nums[k]; l++) {
                    if ((nums[i] + nums[j] + nums[k]) % l === 0)
                        isPrime = false;
                }
                if (isPrime){
                    result++;
                }
            }
        }
    }
    return result;
}

File: 프로그래머스/1/12977. 소수 만들기/소수 만들기.py
def solution(nums):
    result = 0
    temp = 0
    is_prime = True
    
    for i in range(len(nums) - 2):
        for j in range(i + 1, len(nums) - 1):
            for k in range(j + 1, len(nums)):
                temp = nums[i] + nums[j] + nums[k]
                is_prime = True
                for l in range(2, int(temp / 2)):
                    if temp % l == 0: is_prime = False
                if is_prime: result += 1
    return result